### PR TITLE
修复多分类问题

### PR DESCRIPTION
**Pull request by xuqiang**

### CODE REVIEW

File: src/cuBERT/op_out/AdditionalOutputLayer.cpp [hunk 1]
```diff
@@ -13,9 +13,18 @@ namespace cuBERT {
         this->handle = handle;
         this->hidden_size = hidden_size;
         this->num_labels = num_labels;
+        
+        std::cerr << "hidden_size:" << this->hidden_size << " num_labels:" << this->num_labels << std::endl;
 
-        this->output_weights = static_cast<T *>(cuBERT::malloc(sizeof(T) * hidden_size));
-        cuBERT::memcpy(this->output_weights, output_weights, sizeof(T) * hidden_size, 1);
+        if(output_weights != nullptr) {
+            this->output_weights = static_cast<T *>(cuBERT::malloc(sizeof(T) * hidden_size * num_labels * max_batch_size));
```
Comment (levyfan) [inline src/cuBERT/op_out/AdditionalOutputLayer.cpp]: output_weights 不需要乘上 max_batch_size，所以这里只用修复

```
this->output_weights = static_cast<T *>(cuBERT::malloc(sizeof(T) * hidden_size * num_labels));
cuBERT::memcpy(this->output_weights, output_weights, sizeof(T) * hidden_size * num_labels, 1);
```

File: src/cuBERT/op_out/AdditionalOutputLayer.cpp [hunk 2]
```diff
@@ -25,6 +34,9 @@ namespace cuBERT {
         } else {
             this->output_bias = nullptr;
         }
+
+        this->softmax = new Softmax<T>(max_batch_size, num_labels);
```
Comment (levyfan) [inline src/cuBERT/op_out/AdditionalOutputLayer.cpp]: softmax 可以作为后续的 feature 引入进来，本次的 pr 先专注于修复 num_labels 的问题吧。

https://github.com/google-research/bert/blob/d66a146741588fb208450bde15aa7db143baaa69/run_classifier.py#L608

后续的 softmax 可以改成 `_in_compute(size_t batch_size, T *input, T *output_logits, T *output_prob)` 来支持同时输出多个结果变量。

Comment (xuqiang) [reply src/cuBERT/op_out/AdditionalOutputLayer.cpp]: 恩，这个加个类别cuBERT_SOFTMAX可能更合适些
